### ISSUE TITLE
Instruct to sort by Pageviews descending

### ISSUE THREAD

**Issue by dkozii** [browserslist/browserslist-ga-export#16]
Readme has really cool instruction for creating correct CSV-file, but this point can lead to wrong output:
```
3. Sort the report by Browser column ascending alphabetically (arrow in column header facing up).
```

If you has more than 5000 rows in your statistics and do this, you can lose some data for browsers which names are in the end of alphabet. For example, in my case it was more than 7000 rows, so I could've lost about one third of all statistics, because GA only allows to export 5000 rows.

In my opinion, better to sort by pageviews descending, not by browsers.

**Comment by galvarez421** [comment]
Hi @dkoziy, thank you for the feedback!

Given the 5000-row limitation you mentioned, I agree that it would be better to sort the rows by pageviews descending, but the reason the instructions are to sort by browser column ascending is so that the report used by this tool matches as much as it can the report used by browserslist (see https://github.com/browserslist/browserslist-ga/issues/33).

Given the project's goal ("browserslist-ga-export allows you to generate the same browserslist-stats.json that Browserslist-GA would generate..."), my preference is for the instructions to continue indicating that the report should be sorted by browser ascending, but I am definitely open to including a parenthetical instruction or sub-bullet indicating something like "You may sort by Pageviews descending instead to better account for browsers based on their usage, but note that doing this may result in a different custom usage data file than the one that browserslist-ga would generate." Feel free to submit a PR for this kind of change (please request a merge into the `develop` rather than the `master` branch). Granted, the custom usage data file that this tool generates is otherwise still not yet 100% matched with what browserslist-ga-export would generate (#9, #12, etc.), but those are separate issues that we will also hopefully address soon! 

Ideally, browserslist-ga will change the sorting that it uses, in which case this tool will follow suit. I think part of my motivation in asking the questions in https://github.com/browserslist/browserslist-ga/issues/33 was to figure out the rationale before suggesting a change.

**Comment by dkoziy** [comment]
@galvarez421 thanks for such good feedback, now I understand all reasons for initial instruction. 

And I've discovered another possible bug with my suggestion -- sorting by pageviews descending can lead to losing the most used browser data because of #12. I think that it should be fixed before changing something in instructions. I'll take a look at it when I'll have free time, and then we could continue with this README issue.

**Comment by galvarez421** [comment]
Hi @dkozii, the issue in #12 has been fixed as of [v3.0.0](https://github.com/browserslist/browserslist-ga-export/releases/tag/v3.0.0). Would you be interested in editing your pull request per [my comment](https://github.com/browserslist/browserslist-ga-export/pull/17#issuecomment-635064768)? Just checking -- If not, I will plan to make the change myself. Thanks!

**Comment by galvarez421** [comment]
As it turns out, we don't need to worry about the 5,000-row limitation anymore because this tool now allows you to merge multiple pages/exports of a report into a single custom usage data file (see https://github.com/browserslist/browserslist-ga-export/commit/bf033fc0bcd6f49be88cc5a2d4bf2f0498cb5b22).
